django/contrib/admin/static/admin/css/nav_sidebar.css

```diff
@@ -88,3 +88,7 @@
 #nav-sidebar .current-model {
     background: #ffc;
 }
+
+#nav-sidebar + .content {
+    max-width: calc(100% - 24px);
+}
```

Comment:
This can be reverted after 071efb13a2116a2bef638151f293f48642e9549c

Reply:
Removed.